processor/tailsampling/config.go

```diff
@@ -32,21 +32,22 @@ const (
 	// StringAttributeFilter sample traces that a attribute, of type string, matching
 	// one of the listed values.
 	StringAttributeFilter PolicyType = "string-attribute-filter"
-	// RateLimiting allows all traces until the specified limits are satisfied.
-	RateLimiting PolicyType = "rate-limiting"
+	// RateLimitingFilter allows all traces until the specified limits are satisfied.
+	RateLimitingFilter PolicyType = "rate-limiting-filter"
```

Comment:
Do we need the suffix `-filter`? Are their any policies that are not filters? If not I think it would be simpler to just drop it.

Reply:
Added because all other policies have the "-filter" suffix. We can remove it from all other configs instead, I don't have a strong preference here.

Reply:
I vote for removing from all, unless there is a reason for the -filter prefix. @pjanotti @bogdandrutu what do you think?

Reply:
Agreed: the suffix seems unnecessary, I don't recall any reason to have them.

Reply:
SG, updated in https://github.com/open-telemetry/opentelemetry-service/pull/221/commits/39c52bdf50914365d42f6adff2ab1992319fb9a8.